inference-engine/tests/functional/inference_engine/ir_serialization/models/experimental_detectron_detection_output.xml

```diff
@@ -1,38 +1,106 @@
 <?xml version="1.0" ?>
 <!--This is syntetic model created by hand desined only for white-box unit testing-->
-<net name="add_abc" version="10">
-	<layers>
-		<layer id="0" name="A" type="Parameter" version="opset1">
-			<data element_type="f32" shape="1"/>
-			<output>
-				<port id="0" precision="FP32">
-					<dim>1</dim>
-				</port>
-			</output>
-		</layer>
-		<layer id="1" name="DetectionOutput" type="ExperimentalDetectronDetectionOutput" version="experimental">
```

Comment:
we need to test both experimental and opset6 opsets.

Reply:
Done.